baselines/dasha/README.md

```diff
@@ -0,0 +1,118 @@
+---
+title: DASHA: Distributed Nonconvex Optimization with Communication Compression and Optimal Oracle Complexity
```

Comment:
The rendered readme shows an error. I think it's because the `title` here should be between double quotes (`"<DASHA: ....>"`

Reply:
I think I fixed it. The problem was in other places.